### PR TITLE
Added tests for Lock Renewal usecases #103

### PR DESCRIPTION
**Pull request by ramya0820**
## Description

Brief description of the changes made in the PR. This helps in making better changelog
- Added test cases based on discussion around #103

# Reference to any github issues
- #103

### CODE REVIEW

File: test/renewLock.spec.ts [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+  should.equal(Array.isArray(msgs), true);
+  should.equal(msgs.length, 1);
+  should.equal(msgs[0].body, testMessages[0].body);
+  should.equal(msgs[0].messageId, testMessages[0].messageId);
+
+  console.log(`MessageLockedUntil: ${msgs[0].lockedUntilUtc}`);
+
+  // Compute expected initial lock duration
+  currentExpectedLockExpiryTimeUtc.setSeconds(
```
Comment (ramya-rao-a) [inline test/renewLock.spec.ts]: Can we move this to right after `currentExpectedLockExpiryTimeUtc` is declared?
I was initially wondering that the current expected lock expiry time could not be the current time i.e.`new Date()` until I reached this line

File: test/renewLock.spec.ts [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  });
+
+  it("TopicsSubs - Receive a msg using Batch Receiver, wait until its lock expires, completing it now results in error", async function(): Promise<
+    void
+  > {
+    await testErrorOnLockExpiry(topicClient, subscriptionClient);
+  });
+});
+
+async function testLockRenewalHappyCase(
```
Comment (ramya-rao-a) [inline test/renewLock.spec.ts]: Can we move this just before we call it for the first time?
That's the pattern used for the other tests, and consistency helps for future maintenance and readability

File: test/renewLock.spec.ts [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  await senderClient.sendBatch(testMessages);
+
+  const msgs = await receiverClient.receiveBatch(1);
+
+  should.equal(Array.isArray(msgs), true);
+  should.equal(msgs.length, 1);
+  should.equal(msgs[0].body, testMessages[0].body);
+  should.equal(msgs[0].messageId, testMessages[0].messageId);
+
+  console.log(`MessageLockedUntil: ${msgs[0].lockedUntilUtc}`);
```
Comment (ramya-rao-a) [inline test/renewLock.spec.ts]: Console logs are generally not used in tests. Is there any reason you want to keep them?

File: test/renewLock.spec.ts (hunk 1 above)
Comment (ramya0820) [reply test/renewLock.spec.ts]: Agree, updated the variable name too.

**Pull request merged**